### PR TITLE
Add new target 'AOCODARCH7DUAL'

### PR DESCRIPTION
**Pull request by aocodarc**
hello,please help to add new target 'AOCODARCH7DUAL' for SJET.
It has been tested and validated.
thanks for your work!

### CODE REVIEW

File: configs/default/SJET-AOCODARCH7DUAL.config [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+# pin D13: DMA1 Stream 7 Request 30
+dma pin D14 12
+# pin D14: DMA2 Stream 4 Request 31
+dma pin E05 0
+# pin E05: DMA1 Stream 0 Request 105
+dma pin A08 14
+# pin A08: DMA2 Stream 6 Request 11
+
+# feature
+feature -RX_PARALLEL_PWM
```
Comment (haslinghuis) [inline configs/default/SJET-AOCODARCH7DUAL.config:136]: Maybe we can remove this line too. Firmware already rules this out when `RX_SERIAL` feature is defined. Please test without this line.

**Comment by DusKing1** [comment]
Duplicated PR https://github.com/betaflight/unified-targets/pull/603

File: configs/default/SJET-AOCODARCH7DUAL.config (hunk 1 above)
Comment (aocodarc) [reply configs/default/SJET-AOCODARCH7DUAL.config:136]: ok,I'll do it right away.

**Comment by aocodarc** [comment]
> Duplicated PR #603

This account is the new oficial maintainer email of SJET.
That PR has been closed.

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**